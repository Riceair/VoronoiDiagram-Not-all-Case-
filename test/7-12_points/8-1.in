8
396 145
278 390
63 222
310 286
521 503
113 524
208 294
226 140