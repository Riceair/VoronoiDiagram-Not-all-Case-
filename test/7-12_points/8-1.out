P 63 222
P 113 524
P 208 294
P 226 140
P 278 390
P 310 286
P 396 145
P 521 503
E -190172 31873 77.53 374.73
E -51515.5 -102509 159.22 210.25
E 77.53 374.73 154.52 406.53
E 77.53 374.73 159.22 210.25
E 154.52 406.53 262 328.16
E 154.52 406.53 322.12 612.91
E 159.22 210.25 253.62 221.28
E 253.62 221.28 262 328.16
E 253.62 221.28 309.65 189.04
E 262 328.16 430.43 379.98
E 309.65 189.04 504.55 307.92
E 309.65 189.04 3461 -106957.5
E 322.12 612.91 430.43 379.98
E 322.12 612.91 13547 257553.5
E 430.43 379.98 504.55 307.92
E 504.55 307.92 225998.5 -78426
